src/common/components/index.ts

```diff
@@ -2,3 +2,4 @@ export { MarkDownViewerComponent } from './markdownViewer';
 export { PanelComponent, PanelItem } from './panel';
 export { Breadcrumb, BreadcrumbItem } from './breadcrumb';
 export { CounterButton } from './counterButton/counterButton';
+export { SubscriptionManager } from './subscriptionManager/subscriptionManager';
```

Comment:
only used here, not sure if would be worth (is just one file and not expected to grow and only reference here in index)